src/services/state-persistor/state-persistor-theme.mapper.ts

```diff
@@ -0,0 +1,15 @@
+import { ThemeNodeModel } from '@/composables/themes/themes.model'
+import useThemes from '@/composables/themes/themes.composable'
+
+export class StorageThemeMapper {
+  themeToThemeName(theme: ThemeNodeModel | undefined) {
+    return theme?.name || ''
+  }
+
+  themeNameToTheme(themeName: string) {
```

Comment:
I cannot see where this method is used. Also did not understand why it calls `findBgLayerByName`?

Reply:
Totally forgot to remove this part of the code... I don't even know why I made it call `findBgLayerByName`... Thanks for reporting this!